src/cpp/server/server.cc

```diff
@@ -287,12 +287,14 @@ void Server::Wait() {
 }
 
 void Server::PerformOpsOnCall(CallOpBuffer* buf, Call* call) {
-  static const size_t MAX_OPS = 8;
-  size_t nops = MAX_OPS;
-  grpc_op ops[MAX_OPS];
-  buf->FillOps(ops, &nops);
-  GPR_ASSERT(GRPC_CALL_OK ==
-             grpc_call_start_batch(call->call(), ops, nops, buf));
+  if (call->call()) {
```

Comment:
Under what conditions can we get here?

This seems very dangerous: we'll have someone waiting on a completion that will never arrive, transforming a crash into a stuck thread.